### PR TITLE
SAK-32046: Change text 'My Workspace' which still appears

### PR DESCRIPTION
**Pull request by nicholaswilson100**
in My Home > Resources

### CODE REVIEW

File: reference/docs/conversion/sakai_12_mysql_conversion.sql [hunk 1]
```diff
@@ -241,4 +241,16 @@ DROP TABLE SAKAI_POSTEM_STUDENT_DUPES;
 
 ALTER TABLE SAKAI_POSTEM_STUDENT MODIFY COLUMN username varchar(99), DROP INDEX POSTEM_STUDENT_USERNAME_I,
   ADD UNIQUE INDEX POSTEM_USERNAME_SURROGATE (username, surrogate_key);
--- END SAK-15708
\ No newline at end of file
+-- END SAK-15708
+
+-- BEING SAK-32046 -- Change text 'My Workspace' which still appears in My Home > Resources
+UPDATE CONTENT_COLLECTION
+SET BINARY_ENTITY = REPLACE(BINARY_ENTITY, 'Workspace', 'Home     ')
```
Comment (bjones86) [inline reference/docs/conversion/sakai_12_mysql_conversion.sql]: What's the reasoning for the extra whitespace:

`'Home     '`

**Comment by lardo-de-arnaud** [comment]
The patch should refer to "Home" and not "My Home" which is the Oxford specific term

**Comment by lardo-de-arnaud** [comment]
@bjones86 apparently the spaces are because the replacement DB entry must have the same number of bytes as the original (so Matt J says)

**Comment by jonespm** [comment]
Yeah there's a checksum on the BINARY_ENTITY field so it has to be equals. I'm thinking of changing it to something like Workspace->cont.work and doing the translation through the regular system.

**Comment by jonespm** [comment]
Some languages for sure are longer than the 9 characters so it will be a problem to change this directly. Like it looks like PT has Área Pessoal.

This unfortunately may be a complex fix since i18n will still be involved.

**Comment by ern** [comment]
This issue should be resolved in a more i18n fashion, see the JIRA for more info.

**Pull request closed without merging**